lib/tjbot.js

```diff
@@ -199,8 +199,23 @@ TJBot.prototype._setupCamera = function() {
         encoding: 'jpg',
         outputDir: './',
         verticalFlip: this.configuration.see.camera.verticalFlip,
-        horizontalFlip: this.configuration.see.camera.horizontalFlip
+        horizontalFlip: this.configuration.see.camera.horizontalFlip,
+        time: 1
     });
+    
+    // versions of node-raspistill < 0.0.11 don't have the `time` option, so
+    // force it in if we don't find it
+    if (!this._camera.options.hasOwnProperty('time')) {
```

Comment:
`time`, not `timeout`, it's a property on node-raspistill's `AbstractCamera` object